### ISSUE TITLE
Is this Yolo V3 uses darknet framework and darknet cfg?

### ISSUE THREAD

**Issue by eng100200** [tanluren/yolov3-channel-and-layer-pruning#69]
@tanluren hello, is your code uses darknet framework and darknet cfg?

**Comment by zbyuan** [comment]
You can fine-tune on darknet

**Comment by eng100200** [comment]
@zbyuan no, you did not understand, i guess! i mean does the original code use the original darknet model written in C not in pytorch and require CFG parser? is it? if not, then if you know the details please provide the details.
